src/llmq/quorums_instantsend.cpp

```diff
@@ -850,8 +850,10 @@ void CInstantSendManager::HandleFullyConfirmedBlock(const CBlockIndex* pindex)
         UpdateWalletTransaction(p.second->txid, nullptr);
     }
 
-    // Retry all not yet locked mempool TXs and TX which where mined after the fully confirmed block
-    RetryLockTxs(uint256());
+    if (IsNewInstantSendEnabled()) {
```

Comment:
Shouldn't this `if` be inside `RetryLockTxs` itself?

Reply:
Yeah agree, updated PR